docs/connect.rst

```diff
@@ -99,6 +99,35 @@ You can get a PDO connection like this:
    authenticate as the CrateDB superuser, which is ``crate``. The superuser
    does not have a password, so you should omit the ``password`` argument.
 
+CrateDB Cloud
+=============
+
+If you are connecting to CrateDB Cloud you will need to enable the SSL mode 
+by setting the connection's ``CRATE_ATTR_SSL_MODE`` attribute:
+
+.. code-block:: php
+
+   <?php
+	require __DIR__ . '/vendor/autoload.php';
+	use Crate\PDO\PDOCrateDB;
+	$dsn = 'crate:yourcluster.yourdomain.com:4200';
+	$user = 'user1';
+	echo "Enter pwd for the user1 account: ";
+	$password = rtrim(fgets(STDIN));
```

Comment:
Reading the password interactively is the least likely way it will be implemented on a server. Please either hard-code it like `$password = '<SECRET>`, or, alternatively, read it from an environment variable `CRATEDB_PASSWORD`.